components/script/dom/bluetoothremotegattcharacteristic.rs

```diff
@@ -161,10 +158,9 @@ impl BluetoothRemoteGATTCharacteristicMethods for BluetoothRemoteGATTCharacteris
             BluetoothObjectMsg::BluetoothWriteValue => (),
             BluetoothObjectMsg::Error {
                 error
-            } => {
-                println!("{}", error);
-            },
+            } => return Err(Type(error)),
             _ => unreachable!()
         };
+        Ok(())
```

Comment:
move this up to `BluetoothObjectMsg::BluetoothWriteValue => Ok(()),` and remove `;` from line167 so the `return` won't be needed